.github/workflows/e2e.yml

```diff
@@ -19,17 +19,19 @@ jobs:
     strategy:
       matrix:
         go: [stable, tip]
-        platform: [ubuntu-latest-8-cores, windows-latest, macos-latest]
-    runs-on: ubuntu-latest-8-cores
+        platform: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.platform }}
     steps:
       - name: Checkout code
+        if: matrix.go != 'tip' || matrix.platform != 'windows-latest'
         uses: actions/checkout@v2
       - name: Install Go
+        if: matrix.go != 'tip' || matrix.platform != 'windows-latest'
```

Comment:
I'm wondering if it's possible (an if to me) to put this condition at the job level. If these conditions are still [necessary](https://github.com/grafana/xk6-browser/pull/962/files#r1296911621).

Reply:
That's a nice idea. I tried working with `if` at various levels but it only works in each step. It's because `matrix` is evaluated after the `if` at the [job level](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif).